src/main/resources/templates/config/open-poker-config.vm

```diff
@@ -0,0 +1,35 @@
+$webResourceManager.requireResource("com.aprey.jira.plugin.openpoker.open-poker:open-poker-config-resources")
+<html>
+<head>
+    <title>Open poker configuration</title>
+</head>
+<body>
+
+<header class="aui-page-header">
+    <div class="aui-page-header-inner">
+        <div class="aui-page-header-main">
+            <h1>Open poker configuration</h1>
+        </div>
+    </div>
+</header>
+<h2>Permissions</h2>
+<form action="$action.getURL()" class="aui" method="POST">
+    <input type="hidden" id="allowedProjects" name="allowedProjects" value="$action.getAllowedProjectsValue()"/>
```

Comment:
When I submitted the form I got "XSRF Security Token Missing" error. I managed to fix it by adding the hidden input with the token:
```
<input type="hidden" name="atl_token" value="$atl_token" />
```
More info about it could be found here - https://developer.atlassian.com/server/jira/platform/form-token-handling/